### DISCUSSION TITLE
Queries running multiple times

### DISCUSSION THREAD

**Discussion by cpbret** [livewire/livewire#4407] [1. Help]
Basically, livewire components included in a page using @livewire or <livewire.> render twice and cause queries to be run twice. 

I've seen a number of people wrestling with this issue, ([here](https://github.com/livewire/livewire/discussions/4323) and [here](https://stackoverflow.com/questions/69601285/livewire-performing-double-queries)) and the same solution posted, but it seems to involve creating the component as a full-page component.

I have verified running my existing component full-page works without double rendering, but is it possible to do so using a directive - so that it can be added inline?

**Comment by maweitzman** [comment]
@joshhanley Hey Josh, sorry to always tag you in stuff, but you're a guru at it! Any tips for us?

**Reply by joshhanley** [reply]
@maweitzman no worries! I do go through all discussions regularly and try to make sure I respond to them 🙂

**Comment by joshhanley** [comment]
@cpbret the directive shouldn't be causing it to double render but I've not experienced that issue myself as it all seems to work fine for me.

Any chance you could create a small example repo which reproduces the issue with as little code as possible so we can investigate it?

Thanks!
